student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Student is a class that inherits from Person class. Please add the parameters name, age, and parent_permission to the constructor and to the super method. Check out this link to learn more about the inheritance https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.`